docs/wiki/EL/data-structures.md

```diff
@@ -90,15 +90,28 @@ The structure `T` consists of the following:
+- **maxFeePerGas** - the maximum fee per unit of gas willing to be paid for the transaction (including baseFeePerGas and maxPriorityFeePerGas).
 - **from** – The address of the sender, that will be signing the transaction. This must be an externally-owned account as contract accounts cannot send transactions.
 - **to**: Address of an account to receive funds, or zero for contract creation.
 - **value**: amount of ETH to transfer from sender to recipient.
-- **input data** – optional field to include arbitrary data
+- **input data** – optional field to include arbitrary data.
 - **data**: Input data for a message call together with the message signature.
-- **(v, r, s)**: Values encoding signature of a sender. Serves as identifier of the sender
+- **(v, r, s)**: Values encoding signature of a sender. Serves as identifier of the sender.
 
-### TODO: Explain Receipt Trie
+##  Receipt Trie
+
+The Receipt Trie is similar to the Transaction Trie in that it is a block level data structure, and each leaf of the trie represents some RLP-encoded data related to the transaction. However, the Receipt Trie is used to verify that the instructions in each transaction were actually executed.  This verification data is held in the leaf node and contains a few fields:
+- **Status**: The transaction status is either `0` or `1` where `1` indicates a successful transaction and `0` is for a failed transaction.
+- **Gas Used**: Total gas consumed by all previous transactions in the block + the current transaction's gas used.
+- **Logs**: Includes indexed fields called topics for efficient lookups such as addresses and transaction type. Logs also hold non-indexed raw event data.
+- **Logs Bloom**: A 512-byte bloom filter used to quickly search for relevant logs in a block, which allows applications to efficiently check if an address or event signature is included in logs.
+
+Some additional information on how logs bloom is used to allow applications to efficiently check if an address or event signature is included in logs can be found [here](https://medium.com/coinmonks/ethereum-data-transaction-receipt-trie-and-logs-simplified-30e3ae8dc3cf#:~:text=the%20sections%20below.-,Logs%20Bloom,-Assume%20we%20want).
+
+Here is a simple diagram of a Receipt Trie, which follows the Merkle Patricia Trie flow for value lookups.
+![Receipt Tree](../../images/data-structures/receipt-trie.png)
+
+Receipt Tries enable light clients to efficiently verify transaction results provided by full nodes. Since light clients only store block headers, they rely on full nodes to query execution results. To ensure trustlessness, full nodes return not only the transaction results but also Merkle proofs derived from the receiptsRoot stored in the block header. This structure allows light clients to independently verify the legitimacy of the data without storing the full transaction history.
```

Comment:
Receipts are not mainly for light clients but are useful for indexing data without re-executing transactions and regular full nodes download them during snap sync. 

It would be ideal here to refer to some explanation of what is a receipt. It should be part of transaction anatomy page on the wiki but it's missing there :/ 

Also EIP 2718 should be mentioned on this page https://eips.ethereum.org/EIPS/eip-2718#receipts

Reply:
great context @taxmeifyoucan! added commit https://github.com/eth-protocol-fellows/protocol-studies/pull/399/commits/4158aba4faf68c3a77a8005812706a8227ac3b12 to:
- update receipts use case being for indexing and how receipt trie is recreated during snap sync
- added `Receipt` section to the transaction anatomy page
- added section on EIP 2718 updates to tsx + receipts to transaction anatomy page